src/axom/core/tests/core_array.hpp

```diff
@@ -733,6 +733,174 @@ void check_external_view(ArrayView<T>& v)
   EXPECT_EQ(data_ptr, v.data());
 }
 
+#ifdef __CUDACC__
```

Comment:
Does this file need to check guard/check against Umpire's availability?

Reply:
The snippets further down check for Umpire but since these functions use the full set of memory spaces I have added that to the preproc conditional.